src/views/product-downloads-view/index.tsx

```diff
@@ -48,18 +49,14 @@ const ProductDownloadsViewContent = ({
   } = pageContent
   const currentProduct = useCurrentProduct()
   const { currentVersion } = useCurrentVersion()
-  const backToLinkProps = useMemo(
-    () => initializeBackToLink(currentProduct),
-    [currentProduct]
-  )
   const breadcrumbLinks = useMemo(
     () => initializeBreadcrumbLinks(currentProduct, currentVersion),
     [currentProduct, currentVersion]
   )
-  const navData = useMemo(
-    () => initializeNavData(currentProduct),
-    [currentProduct]
-  )
+  const sidebarNavDataLevels = [
```

Comment:
**Future** 📌 

It seems like we're generating this data server-side for other views. This is almost certainly a future optimization, but I wonder if we could move the `sidebarNavDataLevels` generation into `getStaticProps` for this view 🤔 I think `productData` is already available there, which would give us the equivalent of `currentProduct`. Looks like it that might require the creation of a new `views/product-downloads-view/server.ts` file, so feels outside the scope of this PR  💭

Reply:
I was having the same thoughts while working on this! I can create another PR for adding the `views/product-downloads-view/server.ts` file. It'll be even easier to clean up the downloads page components now too with #410 being merged!